week08/movie/src/components/MovieList.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import axios from "axios";
+import Movie from "./Movie";
+
+const MovieLine = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 0px 10px;
+  overflow: auto;
+  scrollbar-width: none;
+`;
+
+class MovieList extends React.Component {
+  state = {
+    isLoading: true,
+    movies: [],
+  };
+  getMovies = async () => {
+    const { url } = this.props;
+    const {
+      data: {
+        data: { movies },
+      },
+    } = await axios.get(`https://yts-proxy.now.sh/list_movies.json${url}`);
+    this.setState({ movies, isLoading: false });
+    console.log(movies);
+  };
```

Comment:
- [x] 2. try-catch 블록을 통한 에러 핸들링
에러를 잡아낼 수 있는 catch 블록도 getMovies 안에 함께 작성해줍니다.
```suggestion
getMovies = async () => {
    try {
      const { url } = this.props;
      const {
        data: {
          data: { movies },
        },
      } = await axios.get(`https://yts-proxy.now.sh/list_movies.json${url}`);

      this.setState({ movies, isLoading: false });
      console.log(movies);
    } catch (error) {
      console.error(error);
      this.setState({ isLoading: false });
    }
  };
```